### PR TITLE
Enh/noid/simplify sidebar in one to one conversations

### LINKED ISSUE #5511: 1:1 chat room empty
I just wanted to chat with Roeland on our instance but if I go into the room it looks like this (he seems to be no longer in the room):

image

Probably he cleaned up his room list on the left but how can I now get back chatting with him. Do I have to leave the room as welll and start from scratch?

It is a bit confusing.

### PR DESCRIPTION
**Pull request by marcoambrosini**
- Remove participants tab from 1to1s;
- Collapse sidebar when switching to 1to1s;

Closes #5511

### CODE REVIEW

**Comment by nickvergessen** [comment]
This is for 23 only

File: src/App.vue [hunk 1]
```diff
@@ -174,6 +175,22 @@ export default {
...
+		 */
+		currentConversation() {
+			return this.$store.getters.conversation(this.token)
+		},
+
+		/**
+		 * Computes whether the current conversation is one to one
+		 * @returns {boolean} The result
+		 */
+		isOnetoOne() {
```
Comment (nickvergessen) [inline src/App.vue:191]:
```suggestion
		isOneToOne() {
```

File: src/App.vue [hunk 2]
```diff
@@ -184,6 +201,15 @@ export default {
 
 			this.setPageTitle(this.getConversationName(this.token), this.atLeastOneLastMessageIdChanged)
 		},
+
+		token() {
+			// Collapse the sidebar if it's a 1to1 conversation
+			if (this.isOnetoOne || BrowserStorage.getItem('sidebarOpen') === 'false') {
```
Comment (nickvergessen) [inline src/App.vue:207]:
```suggestion
			if (this.isOneToOne || BrowserStorage.getItem('sidebarOpen') === 'false') {
```

**Comment by PVince81** [comment]
there's a conflict

File: src/App.vue [hunk 3]
```diff
@@ -184,6 +201,15 @@ export default {
 
 			this.setPageTitle(this.getConversationName(this.token), this.atLeastOneLastMessageIdChanged)
 		},
+
+		token() {
+			// Collapse the sidebar if it's a 1to1 conversation
+			if (this.isOneToOne || BrowserStorage.getItem('sidebarOpen') === 'false') {
```
Comment (PVince81) [inline src/App.vue:207]: some day we'll store these in the vuex store and the vuex store will take care to store them in BrowserStorage

**Review by PVince81 — APPROVED** [review]
:+1:

**Pull request merged**